CHANGELOG.md

```diff
@@ -34,37 +34,45 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - documentation for adding new ingest formats to Datura
 - byebug gem for debugging
 - instructions for installing Javascript Runtime files for Saxon
-- API schema can either be 1.0 or 2.0 (which includes nested fields); 1.0 will be run by default unless 2.0 is specified. Add the following to `public.yml` or `private.yml` in the data repo:
+- API schema can either be the original 1.0 or the newly updated 2.0 (which includes new fields including nested fields); 1.0 will be run by default unless 2.0 is specified. Add the following to `public.yml` or `private.yml` in the data repo:
 ```
 api_version: '2.0'
 ```
 See new schema (2.0) documentation [here](https://github.com/CDRH/datura/docs/schema_v2.md)
-- schema validation with API version 2.0, invalidly constructed documents will not post
+- schema validation with API version 2.0: invalidly constructed documents will not post
 - authentication with Elasticesarch 8.5; add the following to `public.yml` or `private.yml` in the data repo:
 ```
   es_user: username
   es_password: ********
 ```
 - field overrides for new fields in the new API schema
 - functionality to transform EAD files and post them to elasticsearch
+- functionality to transform PDF files (including text and metadata) and post them to elasticsearch
+- limiting `text` field to a specific limit: `text_field` in `public.yml` or `private.yml`
+- configuration options related to Elasticsearch, including `text_limit`  and `es_schema_override` and `es_schema_path` to change the location of the Elasticsearch schema
```

Comment:
Is the first of these two lines supposed to be referencing `text_limit` rather than `text_field`?

And I think "including `text_limit` and" in the second line should be removed as it's explained in the first line and I don't think it relates to ES schema config.

Let me know if I'm misunderstanding though.

Reply:
yes, it's supposed to be `text_limit`. I will make that change.